README.adoc

```diff
@@ -19,20 +19,20 @@
 
 Explore how to access a simple RESTful web service and consume its resources in Java.
 
-
 // =================================================================================================
 // Introduction
 // =================================================================================================
 
 == What you'll learn
 
-You will learn how to access a REST service and deserialize the returned JSON resources containing a
-list of artists and their albums using two different approaches. The first approach consists of using
-the JSON-P API to process the JSON. The second approach focuses on using a JSON binding library like
-Jackson to convert JSON messages into Java objects directly.
+You will learn how to access a REST service, serialize a java object containing a
+list of artists and their albums, and deserialize the returned JSON resources using
```

Comment:
What about, "...and use two different approaches to deserialize the returned JSON resources."?